app/workers/my_anime_list_list_worker.rb

```diff
@@ -1,19 +1,45 @@
 class MyAnimeListListWorker
   include Sidekiq::Worker
 
-  def perform(user_id)
-    kitsu_library(user_id).each do |library_entry_id|
+  def perform(linked_account_id, user_id)
+    kitsu_library(user_id).each do |library_entry|
+      # create the logs here before the next worker
+      library_entry_log = create_log(library_entry, linked_account_id)
+
       MyAnimeListSyncWorker.perform_async(
-        library_entry_id: library_entry_id,
-        method: 'create/update'
+        library_entry_id: library_entry.id,
+        # HACK: I am unable to determine the method
+        # through the initial sync because I don't access their
+        # mal list. For now everything will be a create action,
+        # but at some point we can update to get the correct action.
+        # Same issue with the LibraryEntryLog performed_action
+        method: 'create',
```

Comment:
Like if we _really_ wanted to be nitpicky, we would update the `method` right here:
```
      if media_type == 'anime' && response['watched_status']
      elsif media_type == 'anime'
```
so that we get a 100% accurate `action_performed` but I don't think it matters that much.